source/MaterialXGenOsl/OslSyntax.cpp

```diff
@@ -181,6 +181,7 @@ namespace
     };
 }
 
+const string OslSyntax::BLOCK_VARIABLE_SEPARATOR = ",\n";
```

Comment:
Could we make this a member of ShaderGenerator instead and name it COLON_NEWLINE